src/common/constants.ts

```diff
@@ -108,10 +143,31 @@ export function stringToType(str: string): CardType {
 
 // Parsing.
 
-const shouldUseLocalParser = USE_LOCAL_PARSER_ON_LOCALHOST && typeof window !== 'undefined' && window.location.hostname === 'localhost';  // tslint:disable-line no-typeof-undefined
-const liveParserUrl = '//parser.wordbots.io';
-const stagingParserUrl = '//wordbots-parser-staging.herokuapp.com';
-export const PARSER_URL = USE_STAGING_PARSER ? stagingParserUrl : (shouldUseLocalParser ? `http://localhost:${LOCAL_PARSER_PORT}` : liveParserUrl);
+const LIVE_PARSER_URL = '//parser.wordbots.io';
+const STAGING_PARSER_URL = '//parser-staging.wordbots.io';
+const LOCAL_PARSER_URL = 'http://localhost:8080';
+
+export const PARSER_URL: string = (() => {
+  // tslint:disable-next-line: no-typeof-undefined
+  if (typeof window !== 'undefined') {
```

Comment:
Why is this necessary? If its checking if its running in browser don't we have a helper function for that?